TUM-Dev/NavigaTUM
# feat:component webclient testing

## Proposed Changes (include Screenshots if possible)

- currently, the frontend is mostly tested via e2e tests
- This PR adds component tests

## How to test this PR

1. see that the testcases work both in ci and locally
2. verify manually, that the content is the same

## How has this been tested?

- see above

## Checklist
